METRICS.md

```diff
@@ -95,6 +95,8 @@ Here is an example of grafana dashboard importing.
 |               DiskBuffer               |      worker       | Disk buffers are part of netty used memory, means data need to write to disk but haven't been written to disk.  |
 |             PausePushData              |      worker       |                   PausePushData means the count of worker stopped receiving data from client.                   |
 |       PausePushDataAndReplicate        |      worker       |    PausePushDataAndReplicate means the count of worker stopped receiving data from client and other workers.    |
+|           ActiveShuffleSize            |      worker       |                                     The active shuffle size of peer worker.                                     |
```

Comment:
Actually, this is the shuffle file size of a worker including master replica and slave replica. So the "peer" is not suitable.